config.yaml.example

```diff
@@ -120,6 +120,29 @@ upgrade:
   # grub menu entry, this is the string that will be displayed
   grub-entry-name: Elemental
 
+# configuration used for the 'mount' command
+mount:
+  sysroot: /sysroot # Path to mount system to
+  write-fstab: true # Write fstab into sysroot/etc/fstab
+  overlay:
```

Comment:
I am wondering if this is the right name. I mean if we have `persistent` then probably we should have something like `transient` or `ephemeral`. Otherwise one could think this overlay device setting is for the persistent data.